src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -725,6 +726,24 @@ Engine::LoadExeNetworkImpl(const InferenceEngine::CNNNetwork &network, const std
         conf.batchLimit = static_cast<int>(network.getBatchSize());
     }
 
+    // SSE runtime check is needed for some ATOM machine, which is x86-64 but w/o SSE
+    static Xbyak::util::Cpu cpu;
+    if (cpu.has(Xbyak::util::Cpu::tSSE)) {
+        std::cout << "DenormalsOptMode is set to ON." << std::endl;
+        flush_to_zero(true);
+        denormals_as_zero(true);
+        // if (conf.denormalsOptMode == Config::DenormalsOptMode::DO_On) {
+        //     std::cout << "DenormalsOptMode is set to ON." << std::endl;
+        //     flush_to_zero(true);
+        //     denormals_as_zero(true);
+        // } else if (conf.denormalsOptMode == Config::DenormalsOptMode::DO_Off) {
+        //     std::cout << "DenormalsOptMode is set to OFF." << std::endl;
+        //     flush_to_zero(false);
+        //     denormals_as_zero(false);
+        // }
+        // std::cout << "DenormalsOptMode is not explicitly set and keep." << std::endl;
+    }
+
```

Comment:
The default behavior here is important, considering user may set FTZ DAZ outside OpenVINO and he/she maybe surprised by our default setting on these flags, maybe by default we should do nothing, but in benchmark_app we can enable this feature by default

Reply:
This is actually already how it works now. The default is [DO_KEEP](https://github.com/openvinotoolkit/openvino/pull/11883/files#diff-76c2faa205c3624b84079d81a130db2ddac8513dd1c7df85ea327b51171793beR51). If user do not set this property explicitly, the DO behavior inherit from parent environment where user may set outside OV. Only the value of key is ["YES" or "NO"](https://github.com/openvinotoolkit/openvino/pull/11883/files#diff-bfc4c8727c69c70f65b7b1fb1261169bc09c9b202865cd8048ab892999ba0717R151), we set this accordingly.

Reply:
The code here is hard-code for DLB test temporary. will revert a later time

Reply:
Another concern is, the setup of FTZ DAZ will not take effect immediately until Loadnetwork is called, which is pretty weird from user's perspective, especially when user want to turn of these flags